src/data/modifiers.txt

```diff
@@ -9039,6 +9047,7 @@ Item	green hamhock	Effect: "Just the Best Anapests", Effect Duration: 50
 Item	green-iced sweet roll	Effect: "Sprinkle Sense", Effect Duration: 40
 Item	gregarious ghostling	Free Pull
 Item	grinning ghostling	Free Pull
+# grog nuts
```

Comment:
while we add in blank sections for equipment / effects without effects, I don't think we tend to do the same for food.

I don't think we should at all, really.

Reply:
I agree. I'll remove it